content/docs/capabilities/directory-sync.mdx

```diff
@@ -0,0 +1,54 @@
+---
+id: directory-sync
+title: Directory Sync
+description: Directory Sync in Pomerium Enterprise allows you to import organizational directory data and external data sources you can use in authorization policies.
+keywords: [directory sync, idp, identity provider, groups, directory]
+---
+
+# Directory Sync
+
+**Directory Sync** is the process of synchronizing external directory data from your identity provider into the Enterprise Console. This document discusses how directory sync works in Pomerium and its use cases.
+
+:::enterprise
+
+**Directory Sync** is a Pomerium Enterprise feature. [Contact us](https://www.pomerium.com/enterprise-sales/) to upgrade today.
+
+:::
+
+## Directory sync in the Enterprise Console
+
+To start a directory sync in the Enterprise Console:
+
+1. Go to the **Identity Providers** tab
+1. Select your **Identity Provider**
+1. Next to [**IDP Options**](#idp-options), fill out the required fields
```

Comment:
I think maybe we should add a "see below" link or something like that here. I remember being very confused by these options when I was trying to enable directory sync for testing. I think it might be helpful to have some additional framing here to explain that this varies from provider to provider.